monkey/common/cloud/azure/azure_instance.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import requests
+
+from common.cloud.environment_names import AZURE
+from common.cloud.instance import CloudInstance
+
+LATEST_AZURE_METADATA_API_VERSION = "2019-04-30"
+AZURE_METADATA_SERVICE_URL = "http://169.254.169.254/metadata/instance?api-version=%s" % LATEST_AZURE_METADATA_API_VERSION
```

Comment:
So this is the endpoint all azure instances have access to and it is not likely to change?

Reply:
Yes according to their docs: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service